src/tools/sfdk/command.cpp

```diff
@@ -647,3 +1045,5 @@ std::unique_ptr<Worker> EngineWorker::fromMap(const QVariantMap &data, int versi
     return worker;
 #endif
 }
+
+#include "command.moc"
```

Comment:
fatal error: command.moc: No such file or directory
 #include "command.moc"
          ^~~~~~~~~~~~~
compilation terminated.

I believe this is because Qt does not notice the Q_OBJECT for SetPropertiesTask class, as it is not in a header file, and that's why it doesn't generate the moc file. Not that having class declarations in cpp files is a good habit anyway.

Reply:
you need to run qmake again (make -C src/tools/sfdk qmake)